src/components/split-button/split-button.component.js

```diff
@@ -0,0 +1,244 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withTheme } from 'styled-components';
+import Icon from '../icon/icon';
+import Button from '../button';
+import StyledSplitButtonContainer, { StyledToggleButton } from './split-button.style';
+import StyledSplitButtonChildrenContainer from './split-button-children.style';
+import { validProps } from '../../utils/ether/ether';
+import OptionsHelper from '../../utils/helpers/options-helper';
+import Events from '../../utils/helpers/events';
+import { THEMES } from '../../style/themes';
+
+export class SplitButton extends React.Component {
```

Comment:
same question as above, what do you think about...
`export class SplitButton extends Component`